ci/gpu/build.sh

```diff
@@ -92,9 +92,9 @@ conda list
 
 logger "Build libcudf..."
 if [[ ${BUILD_MODE} == "pull-request" ]]; then
-    $WORKSPACE/build.sh clean libcudf cudf dask_cudf benchmarks tests
+    $WORKSPACE/build.sh clean libcudf cudf dask_cudf benchmarks tests --ptds
 else
-    $WORKSPACE/build.sh clean libcudf cudf dask_cudf benchmarks tests -l
+    $WORKSPACE/build.sh clean libcudf cudf dask_cudf benchmarks tests -l --ptds
 fi
```

Comment:
@dillon-cullinan I assume the build mode is "pull-request" when CI is running against a PR. What is the `else` clause here used?

Reply:
For `branch` builds (merge commits + nightlies), we make sure to build the legacy code as well.

Reply:
Got it. Legacy code is gone so we can likely remove that option, but seems like this is good to go then. Thanks!